unlock all features for new rp2040 version of ploopy mouse

shadow-absorber opened the pull request:
removes the tap dance and key override locks from the rp2040 version of ploopy, mouse while keeping them away from the old ATmega32u4 version so both can use as much of vial as possible without memory overflow

Commit:
remove keyboard revision wide feature locks to let rev1_003 use its full
feature set

Commit:
add feature removals to let keyboard compile small enough to fit

xyzz commented on keyboards/ploopyco/mouse/rev1_002/rules.mk:
```diff
@@ -1,2 +1,9 @@
 # Processor frequency
 F_CPU = 8000000
+
+TAP_DANCE_ENABLE = no
```
changes outside of the vial keymap are not allowed, it will conflict with qmk upstream

shadow-absorber replied:
ah okay... any other way to make it stick only to the rev1_002 version or should shadow just scrap that part all together??

shadow-absorber replied:
fixed it by moving vial keymaps folders and setting the 2 revisions as their own "keyboards"

Commit:
change folder structure and add new keyboard id
fixes vial feature set for old and new ploopy mouse.
this is achieved by moving the keymaps folder into the respective
folders.
also change the keyboard id for the new revision.

xyzz commented on keyboards/ploopyco/mouse/rev1_003/keymaps/vial/rules.mk:
```diff
@@ -0,0 +1,16 @@
+
+# LTO_ENABLE = yes
+
+# VIAL_INSECURE = yes
```
remove commented config lines here and in the other rules.mk including VIAL_INSECURE

xyzz commented on keyboards/ploopyco/mouse/rev1_002/rules.mk:
```diff
@@ -1,2 +1,3 @@
 # Processor frequency
 F_CPU = 8000000
+
```
revert this change

Commit:
remove commented out lines

Commit:
remove added newline

Commit:
remove all comments
